rpm/stackdriver-agent.spec

```diff
@@ -251,6 +252,7 @@ mkdir -p %{buildroot}%{_sysconfdir}/collectd.d/
 
 # *.la files shouldn't be distributed.
 rm -f %{buildroot}%{_libdir}/{collectd/,}*.la
+rm -f %{buildroot}%{_sysconfdir}/collectd.conf
```

Comment:
Is this something we should have been doing all along? Or is this new because you're trying to get rid of the tmpl file?

Reply:
Before, the `.conf` file was regenerated on every restart, so it was throwaway. Now, it's a full fledged file and we need to manage it.

Reply:
SGTM.